src/components/Garden/FilterBar/FilterBar.js

```diff
@@ -70,7 +70,7 @@ const FilterBar = React.memo(
         <TextFilter
           textFilter={proposalTextFilter}
           updateTextFilter={handleTextFilterChange}
-          placeholder="Search by name"
+          placeholder="Search by proposal name"
```

Comment:
Realized that we are not using this File anymore or nay of the files inside the `FilterBar`folder, might have forgotten to delete it.

Could we delete the entire folder?